app/controllers/admin/products_v3_controller.rb

```diff
@@ -209,6 +219,18 @@ def products_bulk_params
       params.permit(products: ::PermittedAttributes::Product.attributes)
         .to_h.with_indifferent_access
     end
+
+    def clone_error_message(error)
+      case error
+      when ActiveRecord::RecordInvalid
+        field_names = error.record.errors.attribute_names
+          .map{ |field_name| VIEW_FIELD_NAME_MAPPER[field_name] }.join(', ')
+
+        field_names.present? ? t('.invalid_fields_error', field_names:) : t('.error')
```

Comment:
I could only think of these fields that may fail product cloning and the user may be able to fix these. If the `field_names` are empty, we display the default error message.
I believe product cloning would fail only on very rare occasions and I'm not sure if it's worth spending more on time investigating all the cases that may fail it and showing the error message to fix it.